docs/source/developers/guide/step_by_step/finding_issues.rst

```diff
@@ -69,6 +69,8 @@ in the comments.
    When you find a GitHub issue you would like to work on, please mention
    your interest in the comment section of that issue; that way we will know
    you are working on it.
+   To assign yourself to the issue type 'take' (and nothing else) in a comment
+   on that issue.
```

Comment:
We have a note for `take` at https://arrow.apache.org/docs/dev/developers/bug_reports.html#issue-assignment .
How about linking to the section instead of writing the same content here?

Reply:
Oh, that's where it is! Sure.

Reply:
(I pushed a change, but I managed to build docs locally, and it does not work as I expected. I need to understand how linking to a subdomain on a seperate page works. `:ref:` might work. I will move to "ready for review" when it is building correctly.